src/utils/normalization.ts

```diff
@@ -68,22 +69,42 @@ export const normalize = (payload: OperationsRepository) => {
             })
           );
 
-          const airkeeper = Object.fromEntries(
-            Object.entries(value.airkeeper).map(([key, value]) => {
+          const airnodeGCP =
+            value.airnode.gcp &&
+            Object.fromEntries(
+              Object.entries(value.airnode.gcp).map(([key, value]) => {
+                if (key === 'gcp') {
+                  const content = Object.fromEntries(Object.entries(value).map(([key, _value]) => [key, '']));
+                  return [key, { ...content }];
+                }
+                return [key, value];
+              })
+            );
+
+          const airkeeperAWS = Object.fromEntries(
+            Object.entries(value.airkeeper.aws).map(([key, value]) => {
               if (key === 'secrets') {
                 const envBuffer = Buffer.from((value as Secrets).content);
                 const content = Object.entries(parse(envBuffer))
                   .map(([key, _value]) => key)
                   .concat([''])
-                  .join('=""\n');
+                  .join('=\n')
+                  .trim();
 
                 return [key, { ...value, content }];
               }
 
               return [key, value];
             })
           );
-          return [key, { airnode, airkeeper }];
+          return [
+            key,
+            {
+              ...value,
+              airnode: { aws: airnodeAWS, ...(value.airnode.gcp && { gcp: airnodeGCP }) },
```

Comment:
The assumption here is that AWS configuration will always exist but GCP might not, right?